### DISCUSSION TITLE
Hack Day: 2024-08-06

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#784] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by mfisher87** [comment]
Picked up integration test work, broke down tasks and split with Luis: https://github.com/nsidc/earthaccess/pull/749

**Reply by danielfromearth** [reply]
Diagramming the test procedure: https://docs.google.com/presentation/d/1K2kt0dvhtNOdQKW5fNnv3DIIraiK5uI7eqZZqjQC-_w/edit?usp=sharing

**Comment by andypbarrett** [comment]
Andy worked on adding material to the User Guide Authentication page.  I will push to the `update-authentications-docs` branch.
